src/components/Item/AppItem/Item.js

```diff
@@ -31,7 +24,7 @@ const getIframeSrc = (appDetails, item, itemFilters) => {
     return iframeSrc
 }
 
-const AppItem = ({ dashboardMode, item, itemFilters }, context) => {
+const AppItem = ({ dashboardMode, item, ouItemFilters }, context) => {
```

Comment:
The fix looks good.
It's been a while, but I think the original idea here was to pass all the filters in `itemFilters` and pass them to the iframe URL.
Apparently only the ou one was implemented, but I'm wondering if we should keep `itemFilters` for when other filters need to be passed in (like period).
Perhaps all filters should already be passed?
Probably not all AppItem widgets support all of them but wouldn't it be more correct to pass all (not sure about the URL query parameters naming)?